dockers/docker-saiserver-cavm/Dockerfile

```diff
@@ -0,0 +1,30 @@
+
+FROM docker-base
+
+RUN apt-get update \
+    && apt-get install -y libsensors4-dev \
+    python \
+    python-dev
+
+COPY deps /root/deps
+
+RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; }; \
```

Comment:
dpkg_apt is not used anymore
Please see this [example](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-database/Dockerfile.template) on how to use SED_DPKG and this [example](https://github.com/Azure/sonic-buildimage/blob/master/rules/docker-database.mk) on how to integrate a docker image into build system